tutorial.md

```diff
@@ -896,10 +879,14 @@ Update your `golem.db` in the table `appconfig` app with ID `801964d531675a235c9
         }
     }
 ```
-5. (optional) Enter a private network by starting all golem nodes with the `protocol_id` option
+4. (optional) Enter a private network by starting all golem nodes with the `protocol_id` option
 ```bash
 golemapp --protocol_id=1337
 ```
+5. Start both golem nodes with the task-api developer mode enabled
+```bash
+golemapp --task-api-dev=1
```

Comment:
When I try this command, terminal gives;
```
Error: --task-api-dev option does not take a value
```

So, seemingly this should be  `golemapp --task-api-dev`

Reply:
Thanks for the heads up! we continued working on this in golemfactory/task-api-docs
Checked the argument and its proper there :)

Closing this PR and will make a new one syncing the 2 docs